packages/wpcom-checkout/src/checkout-labels.ts

```diff
@@ -59,14 +63,26 @@ export function getSublabel( serverCartItem: ResponseCartProduct ): string {
 		}
 	}
 
-	if ( ! isRenewalItem && serverCartItem.months_per_bill_period === 1 ) {
-		return String( translate( 'Billed monthly' ) );
+	if ( isAddOn( serverCartItem ) && ! isRenewalItem ) {
+		return String( translate( 'Add-On' ) );
 	}
 
 	if ( isRenewalItem ) {
 		return String( translate( 'Renewal' ) );
```

Comment:
I did this like the above, because somehow "Add-On Subscription" and "Add-On Renewal" (which would be more consistent with how we label things for plans) sounded really odd to me.